buildlib/pr/io_demo/az-stage-io-demo.yaml

```diff
@@ -79,7 +86,11 @@ steps:
     analyzer_args="-d $(workspace)/${{ parameters.name }}"
     analyzer_args="$analyzer_args --duration ${{ parameters.duration }}"
     analyzer_args="$analyzer_args -t 3"
-    analyzer_args="$analyzer_args ${{ parameters.analyzer_allow_list_args }}"
+    if [[ ${{ parameters.test_analyzer_args }} == '--no-allow-list' ]]; then
```

Comment:
maybe better change here instead:
https://github.com/openucx/ucx/blob/8bcb61f593c9f1f3adbae572728beab37f38acb8/buildlib/pr/io_demo/io-demo.yml#L153

Reply:
It's applicable only for PullRequest , if we launch manually it will be fail ( System.PullRequest.TargetBranch=Null) , for set tag too, but currently it test doesn't use in tag action